### ISSUE TITLE
CommandBar: Not accessible when button on which command bar has focus gets disabled dynamically.

### ISSUE THREAD

**Issue by adharpicstore** [OfficeDev/office-ui-fabric-react#1674]
### Bug Report

- __Package version(s)__: 2.20.2
- __Browser and OS versions__: All Browsers

#### Priorities and help requested (not applicable if asking question):

Requested priority: High

Products/sites affected: www.visualstudioonline.com

#### Describe the issue:
Command Bar becomes inaccessible when button on which command bar has focus get disabled dynamically.

Repro step:

1. Get focus on command Bar button
2. Disable the button dynamically through toggle.
3. Try to access the command bar

Command bar becomes inaccessible.

#### Actual behavior:

Command bar becomes keyboard inaccessible.

#### Expected behavior:

Keyboard focus should go on next active element in commandBar.

### If applicable, please provide a codepen repro:

http://codepen.io/adsurang/pen/Qpqbdp
